lib/ansible/modules/files/find.py

```diff
@@ -46,7 +46,17 @@
         version_added: "2.5"
     contains:
         description:
-            - One or more regex patterns which should be matched against the file content.
+            - match a regex pattern from the beginning of a line, in a file.
+            - if contains_anywhere is set to yes, match a regex pattern against line in a file.
```

Comment:
Wrap `contains_anywhere` in `C(contains_anywhere)` so it gets code formatted in the HTML docs.

Reply:
Also capitalize the first word in both of these sentences, please.